src/app/component/shared/header/header-dropdown/header-dropdown.component.ts

```diff
@@ -54,4 +55,16 @@ export class HeaderDropdownComponent implements OnInit {
   constructor() {}
 
   ngOnInit() {}
+
```

Comment:
this looks like it is reused. Move it somewhere appropriate and remove copies in other files